### PR TITLE
Added VMware Tanzu Dotnet Buildpack documentation.

### PR DESCRIPTION
**Pull request by miransar**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        </td>
+
+        <td>
+            AWS, GCP, Azure, and vSphere
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+## Install the buildpack [#install]
```
Comment (akristen) [inline src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx:110]: Needs a better title; this is a stand in so we know where the action starts

File: src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+The New Relic Dotnet Extension Buildpack for VMware Tanzu can be installed via the tile in Ops Manager. Alternatively, you can extract the `.Pivotal` file and install individual extension buildpacks using the cf Command Line Interface (CLI) command `cf create-buildpack`.
+
+After you start monitoring your apps, you can set alerts based on any metrics that are collected by Dotnet agents using the New Relic alerting subsystem.
```
Comment (akristen) [inline src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx:114]: Mention of alerts should be moved to what's next? or deleted

File: src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+The New Relic Dotnet Extension Buildpack for VMware Tanzu installs one or more of the following buildpacks depending on the tile configuration (total of 8 extension buildpacks):
```
Comment (akristen) [inline src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx:116]: I'm not sure where this info belongs. It seems relelvant and the closest to an 'overview' type content. I can see it living near the section of content where we gesture to installing some buildpacks but not all, too

File: src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+* New Relic Dotnet Core Extension Cached Buildpack for Dotnet Core Applications running on Ubuntu Jammy 1.* in disconnected (isolated) VMware Tanzu deployments. This is for support of air-gapped environments where there is no access to the outside world.
+* 3 New Relic HWC Extension Buildpacks for Dotnet Framework Applications running on Windows 2019 stacks. This extension buildpack is non-cached.
+* 3 New Relic HWC Extension Cached Buildpacks for Dotnet Framework Applications running on Windows 2019 stacks in disconnected (isolated) VMware Tanzu deployments. This is for support of air-gapped environments where there is no access to the outside world.
+
+All buildpacks use the multi-buildpack approach of Cloud Foundry and require either the standard Dotnet Core buildpack or HWC buildpack to be specified as the last buildpack in the buildpack chain, either in the app manifest or in the `cf push` command line.
+
+<Callout variant="important">
+    The cached version of this extension buildpack for both Dotnet Core and Dotnet Framework contains New Relic Dotnet Agents version `9.1.1`
```
Comment (akristen) [inline src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx:126]: Does this mean that this buildpack is only compatible with this agent version? We should x-check what our current supported DOTNET agent is.

File: src/nav/cloudfoundry.yml [hunk 5]
```diff
@@ -1,7 +0,0 @@
-title: Cloudfoundry
```
Comment (akristen) [inline src/nav/cloudfoundry.yml:1]: Removed this as we have these docs housed in our integrations section currently

File: src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx (hunk 4 above)
Comment (miransar) [reply src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx]: No, compatibility is taken care in the integration itself.

**Pull request merged**